vendor/modules.txt

```diff
@@ -174,7 +174,7 @@ github.com/blang/semver
 github.com/boombuler/barcode
 github.com/boombuler/barcode/qr
 github.com/boombuler/barcode/utils
-# github.com/davecgh/go-spew v1.1.1
+# github.com/davecgh/go-spew v1.1.0
```

Comment:
I think these want to be removed?

Reply:
Its still an indirect dependency unfortunately. Deep in:

```
aws/aws-sdk-go
jmespath/go-jmespath
davecgh/go-spew
```